.github/CODEOWNERS

```diff
@@ -305,7 +305,7 @@
 #/<NotInRepo>/                                                     @TravisCragg-MSFT @hilaryw29
 
 # PRLabel: %Confidential Ledger
-/sdk/confidentialledger/                                           @christothes
+/sdk/confidentialledger/                                           @christothes @PallabPaul
```

Comment:
Can we also add a `ServiceLabel` entry similar to the one that you added for Python?

Reply:
Added!